pkg/gateway/server/oauth_apps.go

```diff
@@ -662,9 +671,10 @@ func (s *Server) storeSlackTrigger(apiContext api.Context, tokenResp types.OAuth
 			Namespace: apiContext.Namespace(),
 		},
 		Spec: v1.SlackTriggerSpec{
```

Comment:
@ibuildthecloud this is so that when slack receiver is deleted, the trigger will be deleted too